AI/vllm-deployment/README.md

```diff
@@ -36,31 +36,40 @@ This example demonstrates how to deploy a server for AI inference using [vLLM](h
 
 ## Detailed Steps & Explanation
 
-1. Ensure Hugging Face permissions to retrieve model:
+1. Create a namespace. This example uses `vllm-example`, but you can choose any name:
+
+```bash
+kubectl create namespace vllm-example
+```
+
+2. Ensure Hugging Face permissions to retrieve model:
```

Comment:
FYI" you can just use `1.` and markdown will render a ordered list for you, so that you don't need to update the number whenever you add/remove something from the list. 

e.g. 
```
1. xxx 
1. xxx
```
is rendered as  
1. xxx 
1. xxx